install-and-configure/install/multi-cluster/federated-etl/aggregator.md

```diff
@@ -184,6 +187,19 @@ data to be ingested.
 
 ## Troubleshooting Aggregator
 
+### Understanding the state of the Aggregator
+
+```txt
+https://kubecost.myorganization.com/model/debug/orchestrator
+```
+
+This is a common endpoint for debugging the state of the Aggregator. It returns a JSON response with details such as:
+
+* What is Aggregator's current state? If ingesting, it is downloading & processing ETL files into the DB. If deriving it is pre-computing commonly used queries into saved tables.
+* What is the ingestion progress for each of the data types? (e.g. asset, allocation, cloud cost, etc.)
+* How fresh is my read database? An epoch timestamp can be found in the `readDBPath`.
+* How frequently is my newly ingested data being promoted into the read database? Reference `currentBucketRefreshInterval`.
```

Comment:
@jessegoodier Thoughts? Lmk if I'm missing anything.

Reply:
nice work. I can't think of anything else to add at the moment.